src/kohlrahbi/table_header.py

```diff
@@ -0,0 +1,170 @@
+"""
+This module contains the TableHeader class.
+"""
+
+from enum import StrEnum
+from typing import Dict, List, cast
+
+from attrs import define
+from docx.table import _Cell  # type:ignore[import]
+from docx.text.paragraph import Paragraph  # type:ignore[import]
+
+
+class HeaderSection(StrEnum):
+    """
+    Enum for the different sections of the table header
+    """
+
+    BESCHREIBUNG = "beschreibung"
+    KOMMUNIKATION_VON = "kommunikation von"
+    PRUEFIDENTIFIKATOR = "pruefidentifikator"
+
+
+# TODO this function is also used in the seed.py module
+#  -> move it to a common module to avoid code duplication and circular imports
+def get_tabstop_positions(paragraph: Paragraph) -> List[int]:
+    """Find all tabstop positions in a given paragraph.
+
+    Mainly the tabstop positions of cells from the middle column are determined
+
+    Args:
+        paragraph (Paragraph):
+
+    Returns:
+        List[int]: All tabstop positions in the given paragraph
+    """
+    tabstop_positions: List[int] = []
+    for tabstop in paragraph.paragraph_format.tab_stops:
+        tabstop_positions.append(tabstop.position)
+    return tabstop_positions
+
+
+def create_mapping_of_tabstop_positions(
```

Comment:
Is this a good place for this function or should we put it also into the class method `from_header_cell` ?